.github/actions/cache-pip/action.yml

```diff
@@ -5,7 +5,7 @@ runs:
   steps:
     - uses: ./.github/actions/py-cache-key
       id: py-cache-key
-    - uses: actions/cache@v3
+    - uses: actions/cache@704facf57e6136b1bc63b828d79edcd491f0ee84 # v3.3.2
```

Comment:
Is it possible to only update a single action as a trial (we can use a short interval for dependabot. I'm curious what the update process looks like.

Reply:
If that succeeds, we can update other actions.

Reply:
Dependabot is configured to monitor all workflows, so it's not possible to update a single Action. However, we could do the change from major-version pins to hash pins for a single Action and see how that works.

That being said, please take a look at this [example PR](https://github.com/pnacht/libarchive/pull/2/files). The current setting for mlflow would lead to PRs that are identical to this one: one PR updating all Actions with new versions.